ipfs-cluster-service/main.go

```diff
@@ -225,7 +239,8 @@ func main() {
 
 func run(c *cli.Context) error {
 	if c.Bool("init") {
-		initConfig(c.Bool("force"))
+		initConfig(c.Bool("force"), c.Bool("generate-secret"),
```

Comment:
generate-secret and env-swarm-secret are not flags in the main application so just use `false, ""`.